src/Lecture3.hs

```diff
@@ -137,13 +143,14 @@ data List1 a = List1 a [a]
 
 -- | This should be list append.
 instance Semigroup (List1 a) where
-
+    List1 x xs <> List1 y ys = List1 x (xs <> (y : ys))
 
 {- | Does 'List1' have the 'Monoid' instance? If no then why?
 
+
 instance Monoid (List1 a) where
+    mempty = List1 mempty
```

Comment:
Unfortunately, this implementation won't satisfy laws "Right identity" and "Left identity" 😞